paimon-common/src/main/java/org/apache/paimon/format/FormatReaderFactory.java

```diff
@@ -31,6 +31,11 @@ public interface FormatReaderFactory extends Serializable {
 
     RecordReader<InternalRow> createReader(FileIO fileIO, Path file) throws IOException;
```

Comment:
Can you introduce a `Context` for `FormatReaderFactory`?

Reply:
done